exercises/diamond/README.md

```diff
@@ -38,18 +38,16 @@ C···C
 ··A··
 ```
 
-Diamond for letter 'E':
+Diamond for letter 'D':
```

Comment:
I don't think that this change is part of the problem-specifications :slightly_smiling_face: